src/elements/hx-search/_shadow.scss

```diff
@@ -1,5 +1,7 @@
-@import (reference) "mixins";
-@import "../_base";
+@import "vars";
```

Comment:
@CITguy, you're making the changes to `<hx-search>` Shadow DOM.

Noting here for PR #592 `<hx-search>` refactor.